frontend/wasted-app/src/api/food.ts

```diff
@@ -27,3 +27,13 @@ export const getAllFood = async (sortObject?: FoodSortObject): Promise<Food[]> =
     return []
   }
 }
+
+export const addNewFood = async (food: Food) => {
+  const data = await fetch(`${WASTED_SERVER_URL}/Food`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    body: JSON.stringify(food)
+  })
```

Comment:
I think this function should return null or some error if fetching endpoint returns any error code

Reply:
Done! We could use some alert library to make this smoothier for the users. For example: https://github.com/acaziasoftcom/react-native-sweet-alert